src/device/pnet_api.c

```diff
@@ -45,16 +43,12 @@ int pnet_init_only (
       return -1;
    }
 
-   /* Initialize everything (and the DCP protocol) */
-   /* First initialize the network interface */
-   net->eth_handle =
-      pnal_eth_init (p_cfg->if_cfg.main_port.if_name, pf_eth_recv, (void *)net);
```

Comment:
Remove `#define pnal_eth_init  mock_pnal_eth_init` also

Reply:
Done, second push below is a rebase on master